kolibri/auth/models.py

```diff
@@ -78,6 +80,18 @@ def get_full_name(self):
     def get_short_name(self):
         return self.first_name
 
+    def has_perm(self, perm, obj=None):
+        return _user_has_perm(self, perm, obj)
```

Comment:
This looks awkwardly non DRY :) And if `obj=None` is treated as a kwarg, it should be treated like-wise below?